plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -472,8 +472,8 @@ class MarathonStore extends GetSetBaseStore {
     const groups = new ServiceTree(data);
 
     const apps = groups.reduceItems(function(map, item) {
```

Comment:
Well, if you know how to explain what happens in this function please do so :) on top of reducing `groups` that produces `apps` we then pass apps into `CompositeState.addMarathonApp` where we augment `frameworks` with the data from `apps`.